libs/platform/src/lib/components/action-bar/action-bar.module.ts

```diff
@@ -1,12 +1,12 @@
-import { NgModule } from '@angular/core';
-import { CommonModule } from '@angular/common';
-import { ActionBarComponent } from './action-bar.component';
-import { FormsModule } from '@angular/forms';
-import { FundamentalNgxCoreModule } from '@fundamental-ngx/core';
-import { ActionBarActionsComponent } from './action-bar-actions/action-bar-actions.component';
-@NgModule({
-    declarations: [ActionBarComponent, ActionBarActionsComponent],
-    imports: [CommonModule, FormsModule, FundamentalNgxCoreModule],
-    exports: [ActionBarComponent, ActionBarActionsComponent]
-})
-export class ActionBarModule {}
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { ActionBarComponent } from './action-bar.component';
+import { FormsModule } from '@angular/forms';
+import { FundamentalNgxCoreModule } from '@fundamental-ngx/core';
+
+@NgModule({
+    declarations: [ActionBarComponent],
+    imports: [CommonModule, FormsModule, FundamentalNgxCoreModule],

```

Comment:
It's still there, you don't need to import everything

Reply:
removed these.